alf/metrics/metrics.py

```diff
@@ -213,17 +230,24 @@ def _update_accumulator_(acc, val):
 
         alf.nest.map_structure(_update_accumulator_, self._accumulator, values)
```

Comment:
Actually I feel that self._accumulator should be plain tensors instead of buffers, because if we run from a checkpoint, although the metric buffers should be restored, the accumulators should be reset to zero instead of being restored, since all envs start from new episodes.

Reply:
Fixed in #1083